netplan/cli/commands/try_command.py

```diff
@@ -47,13 +47,24 @@ def __init__(self):
         self._config_manager = None
         self.t_settings = None
         self.t = None
+        self._rootdir = '/'
```

Comment:
(nitpick) you could write this slightly more compact as:
```
self._rootdir = os.environ.get('DBUS_TEST_NETPLAN_ROOT', '/')
```
but of course it's debatable if it's also more readable - up to you of course :)